packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -34,3 +42,26 @@ export function showCLINotInstalledMessage() {
 export function isSFDXContainerMode(): boolean {
   return process.env.SFDX_CONTAINER_MODE ? true : false;
 }
+
+export function disableCLITelemetry() {
+  GlobalCliEnvironment.environmentVariables.set(
+    ENV_SFDX_DISABLE_TELEMETRY,
+    'true'
+  );
+}
+
+export async function isCLITelemetryAllowed(
+  projectPath: string
+): Promise<boolean> {
+  if (isCLIInstalled()) {
+    const forceConfig = await new ForceConfigGet().getConfig(
+      projectPath,
+      SFDX_CONFIG_DISABLE_TELEMETRY
+    );
+    const disabledConfig =
+      forceConfig.get(SFDX_CONFIG_DISABLE_TELEMETRY) || 'false';
+    const isDisabled = disabledConfig === 'true' ? true : false;
+    return !isDisabled;
```

Comment:
I find this piece a little hard to read. Would something like this be simpler?

return forceConfig.get(SFDX_CONFIG_DISABLE_TELEMETRY) ? !forceConfig.get(SFDX_CONFIG_DISABLE_TELEMETRY) : true;